lms/templates/instructor/instructor_dashboard_2/cohort-editor.underscore

```diff
@@ -29,11 +29,11 @@
     </header>
 
     <ul class="wrapper-tabs">
-        <li class="tab tab-manage_students is-selected" data-tab="manage_students"><a href="#"><%- gettext("Manage Students") %></a></li>
+        <li class="tab tab-manage_students is-selected" data-tab="manage_students"><a href="#"><span class="sr"><%- gettext('Selected tab') %> </span><%- gettext("Manage Students") %></a></li>
```

Comment:
Overall it is probably not a bad idea to do, however, I strongly suspect we're very inconsistent across the platform. I hesitate to recommend a strategy right now because I don't know what the best thing to do is, and I don't have an RTL native speaker I can easily pose the question to. In the interest of moving forward, perhaps we just merge as-is and we address RTL issues holistically as they come up (eg, when we actually go live with Arabic on edx.org), and I can draft more best-practices.